reference/test/solver/batch_bicgstab_kernels.cpp

```diff
@@ -254,8 +254,9 @@ TEST(BatchBicgstab, CoreCanSolveCsrWithoutScaling)
             .with_default_max_iterations(maxits)
             .with_default_residual_tol(tol)
             .with_tolerance_type(gko::stop::batch::ToleranceType::relative)
-            .with_preconditioner(
-                gko::preconditioner::BatchJacobi<T>::build().on(exec))
+            .with_preconditioner(gko::preconditioner::BatchJacobi<T>::build()
```

Comment:
Same question here.Is this change necessary ? Shouldn't the auto-detect work, or are there any convergence issues ?

Reply:
Changed this back to block jacobi.